src/index.css

```
@@ -30,6 +30,7 @@ body {
   margin: 0;
   display: flex;
   place-items: center;
+  font-size: 0.875rem;
```

```suggestion
  font-size: 14px;
```

예측하기 쉽게 px로 적으면 좋을 것 같습니다.

ex) 아래는 2개의 차이입니다. 해당 내용 다음에 어떤건지 공부해서 공유해주시면 좋을 것 같아요
- 16 * 0.875 * ???
- 14 * ???

---

넵 공부해가겠습니다 px로 수정했습니다[9d39e7f](https://github.com/ToyMakers/todo-mark/pull/11/commits/9d39e7fac3d8b5cd753be94a1e480ea45d079c4f)